src/app/layout/color.scss

```diff
@@ -0,0 +1,5 @@
+$green:  #1CC29F;
+$white : #ffff;
+$black : rgb(0 0 0 / 12%);
+$inp-color : #ccc;
+$warning : rgb(221, 49, 49)
```

Comment:
Use better naming conventions for variables, here $inp-color doesn't explain which type of color it is. is it grey, light grey or medium-gray?